from django.test import TestCase

# тесты были удалены после большого обновления моделей
